src/vm/eetwain.cpp

```diff
@@ -5169,7 +5169,11 @@ OBJECTREF EECodeManager::GetInstance( PREGDISPLAY    pContext,
     if (info.ebpFrame)
     {
         _ASSERTE(stackDepth == 0);
+#if defined(_TARGET_X86_) && defined(FEATURE_PAL)
+        taArgBase = GetCallerSp(pContext) - 2 * sizeof(TADDR);
+#else
         taArgBase = *pContext->GetEbpLocation();
```

Comment:
Below line (5180 ?) has direct access to `pContext->SP`
```
taArgBase =  pContext->SP + stackDepth;
```
Would it be better to use like this?
```
taArgBase = pContext->SP - 2 * sizeof(TADDR);
```